puppet/ha-puppet/const.js

```diff
@@ -19,6 +19,16 @@ export const hassUrl = isAddOn
 export const hassToken = options.access_token;
 export const debug = false;
 
+export const getChromePath = (is_addon) => {
+  if (is_addon) {
+    return "/usr/bin/chromium";
+  } else if (process.platform === "darwin") {
+    return "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome";
+  } else {
+    return "/usr/bin/google-chrome";
```

Comment:
I guess we can also keep it set to `undefined` so it uses the built-in Puppeteer one.